### PR TITLE
Only index xml OCR w/ a role of "transcription".

### PR DESCRIPTION
**Pull request by jkeck**

### CODE REVIEW

File: app/models/full_text_parser.rb [hunk 1]
```diff
@@ -55,7 +55,8 @@ def alto_xml_string_content(content)
   def xpath
     [
       "//contentMetadata/resource/file[@id=\"#{druid}.txt\"]",
-      "//contentMetadata/resource[@type='page']/file[@mimetype='application/xml' or @mimetype='application/alto+xml']"
+      "//contentMetadata/resource[@type='page']/file[@role='transcription']\
+[@mimetype='application/xml' or @mimetype='application/alto+xml']"
```
Comment (jkeck) [inline app/models/full_text_parser.rb]: ¯\\_(ツ)_/¯ on this indentation

Comment (cbeer) [reply app/models/full_text_parser.rb]: I'd vote to just raise the line length.

Comment (jkeck) [reply app/models/full_text_parser.rb]: Hmm, I think it'll end up being pretty long, what about just ignoring it for this method?

**Comment by mejackreed** [comment]
Let's wait just a bit on this until we can resolve the data in prod?

cc @cbeer

File: app/models/full_text_parser.rb (hunk 1 above)
Comment (cbeer) [reply app/models/full_text_parser.rb]: 👍

**Comment by cbeer** [comment]
Data updates should be in place.

**Pull request merged**